AzureCalling/Controllers/BottomDrawerViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class BottomDrawerViewController: UIViewController {
+
+    // MARK: Properties
+
+    weak var tableViewDelegate: UITableViewDelegate?
+    var tableViewDataSource: UITableViewDataSource?
+    var tableView: UITableView!
```

Comment:
Agree with the initialization, just that the properties still need to be optional since we also need to add the following init:
```swift
required init?(coder aDecoder: NSCoder) {
    super.init(coder: aDecoder)
}
```